python/lsst/sims/maf/metrics/baseMetric.py

```diff
@@ -86,9 +86,9 @@ class BaseMetric(object):
     colRegistry = ColRegistry()
     colInfo = ColInfo()
 
-    def __init__(self, col=None, metricName=None, maps=None, units=None,
+    def __init__(self, col='metricdata', metricName=None, maps=None, units=None,
                  metricDtype=None, badval=-666,
-                 plotDict=None, displayDict=None):
+                 plotDict=None, displayDict=None, summaryStatList=None):
```

Comment:
Document what summaryStatList has to be. (I take it, it should be a list of instantiated summary statistic metrics). Gahh. 
We're going to have to update our description of a "metric", you know. It's not just a metric, it's the metric plus the associated plotting dictionary, display information, and summary statistics. Bleeh. Makes a 'Metric' not as simple as it used to be. Maybe there ought to be a class above 'metric' which contains a 'metric' and this additional information. It could then also hold a reference to the slicer used for that metric, maybe the metric values itself. I guess then it's a sliceMetric.
Actually - why did we have to push this plotting dict, etc. into the metric in the first place? Was it because of the old driver design and limitations from pex_config? Why are we NOT just associating them at the sliceMetric level?